octobot_trading/exchanges/abstract_exchange.py

```diff
@@ -228,6 +233,23 @@ async def get_order(self, order_id: str, symbol: str = None, **kwargs: dict) ->
         """
         raise NotImplementedError("get_order is not implemented")
 
+    async def _default_get_order_retry_func(self, order):
+        return order is not None
+
+    async def get_order_with_retry(self, order_id: str, symbol: str = None, retry_func=None, **kwargs: dict) -> dict:
+        t0 = time.time()
+        retry_func = retry_func or self._default_get_order_retry_func
+        while time.time() - t0 < self.GET_ORDER_MAX_RETRY_DELAY:
+            fetched_order = await self.exchange_manager.exchange.get_order(order_id, symbol, **kwargs)
+            if await retry_func(fetched_order):
```

Comment:
shouldn't we call the check method (`retry_func`) `check_order_validness` or something like this? Because it's not a retry function

Reply:
right